src/client/components/share-button/share-button.vue

```diff
@@ -0,0 +1,286 @@
+
+      this.shareButtonsAreVisible = !this.shareButtonsAreVisible
+    },
+
+    showSnackbar () {
+      this.snackbarIsVisible = true
+      window.setTimeout(() => this.snackbarIsVisible = false, 2000)
+    },
+
+    hideSharingButtons () {
+      this.shareButtonsAreVisible = false
+    },
+
+    handleClick (plataform) {
+      this.$emit('shared', plataform)
+      
+      this.hideSharingButtons()
+    },
+
+    copyToClipboard () {
+      const el = document.createElement('textarea')
+      el.value = this.url
+      document.body.appendChild(el)
+      
+      el.select()
+      document.execCommand('copy')
+      document.body.removeChild(el)
+      
+      this.showSnackbar()
+    },
+  },
+  mounted () {
+    if (navigator.userAgent.match(/ipad|ipod|iphone/i)) {
+      this.copyToClipboardIsVisible = false
+    }
+  }
+}
+</script>
+
+<style>
+@import '../app-core/index.css';
+
+.snackbar {
+  position: absolute;
+  bottom: -275px;
+  left: -215px;
+  background-color: var(--text-color);
+  text-align: center;
+  color: var(--background-color);
+  font-size: var(--font-size-small);
+  padding: .5rem;
+  width: 200px;
+  opacity: 0;
+  transition: all .5s ease-in-out;
+  border-radius: 30px;
+}
+
+.snackbar-visible {
+  opacity: 1;
+}
+
+.social-share {
+  position: absolute; 
+  right: var(--spacing-default);
+  top: calc(var(--app-header-mobile-height) + var(--spacing-default));
+  z-index: 1;
+  height: 40px;
+  width: 40px;
+}
+
+@media screen and (min-width: 640px) {
+  .social-share {
+    top: calc(var(--app-header-height) + var(--spacing-default));
+  }
+}
+
+.social-share__button {
+  position: absolute; 
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 512 512' width='512' height='512'%3E%3Cpath d='M128 174.829c-46.48 0-84.293 37.813-84.293 84.293S81.52 343.415 128 343.415s84.293-37.813 84.293-84.293S174.48 174.829 128 174.829zm0 131.122c-25.825 0-46.829-21.005-46.829-46.829s21.005-46.829 46.829-46.829 46.829 21.005 46.829 46.829-21.004 46.829-46.829 46.829zM384 0c-46.48 0-84.293 37.813-84.293 84.293 0 46.48 37.813 84.293 84.293 84.293s84.293-37.813 84.293-84.293C468.293 37.813 430.48 0 384 0zm0 131.122c-25.825 0-46.829-21.004-46.829-46.829S358.176 37.464 384 37.464s46.829 21.004 46.829 46.829c0 25.824-21.004 46.829-46.829 46.829zM384 343.415c-46.48 0-84.293 37.813-84.293 84.293S337.52 512 384 512s84.293-37.813 84.293-84.293S430.48 343.415 384 343.415zm0 131.122c-25.825 0-46.829-21.004-46.829-46.829s21.005-46.829 46.829-46.829 46.829 21.004 46.829 46.829c0 25.824-21.004 46.829-46.829 46.829z' fill='%23FFF'/%3E%3Cpath fill='%23FFF' d='M170.2875231 210.3301817l157.3471845-101.7687935 20.3466984 31.4585208-157.3471845 101.7687935zM170.2383304 309.8150712l20.3948572-31.4239644 152.980344 99.287672-20.3948572 31.4239644z'/%3E%3C/svg%3E");  background-size: 50%;
+  background-position: center center;
+  background-repeat: no-repeat;
+  top: 0; 
+  left: 0; 
+  z-index: 10;
```

Comment:
When you open the app menu on mobile, the share button is visible. I think the menu should be on top.